tests/update/updates_packagekit_kde.pm

```diff
@@ -51,7 +51,13 @@ sub run {
             # the assert_screen, record a soft failure
             wait_still_screen;
             if (match_has_tag('updates_none')) {
-                if (check_screen 'updates_none', 30) {
+                #Check if the applet reports that a restart is needed.
+                if (match_has_tag('updates_installed-restart')) {
+                    power_action 'reboot', textmode => 1;
```

Comment:
Can this press on the button in the notification instead? That way that's also tested

Reply:
I agree that this is better but since this test is blocking Leap 15.2 updates I opted for not increasing test coverage that much and just to mirror what updates_packagekit_gpk does. This module is due for an overhaul anyway since it fails regularly and hasn't been meaningfully changed since Sep 14, 2018.

Reply:
Changed!